go/teams/teams.go

```diff
@@ -820,10 +820,10 @@ func (t *Team) inviteSBSMember(ctx context.Context, username string, role keybas
 	return keybase1.TeamAddMemberResult{Invited: true}, nil
 }
 
-func (t *Team) InviteSeitan(ctx context.Context, role keybase1.TeamRole, label keybase1.SeitanIKeyLabel) (ikey SeitanIKey, err error) {
+func (t *Team) InviteSeitan(ctx context.Context, role keybase1.TeamRole, label keybase1.SeitanKeyLabel) (ikey SeitanIKeyV2, err error) {
 	t.G().Log.Debug("team %s invite seitan %v", t.Name(), role)
 
-	ikey, err = GenerateIKey()
+	ikey, err = GenerateIKeyV2()
```

Comment:
We might want to leave ourselves a way to still generate seitanv1 tokens and have them in integration tests

Reply:
hmm yeah we probably do, i'll bring back the systests for v1 as well

Reply:
thanks. it will be a while before we can phase out v1 completely